### PR TITLE
Add Wazuh agent OS check issue template

### PR DESCRIPTION
**Pull request by mjcr99**
|Related issue|
|---|
|https://github.com/wazuh/wazuh/issues/23524|

## Description

Hi team,

this PR adds a new issue template to quickly open an issue for Agent OS check. The visualization of this new template is as follows:

image

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/agent-os-check.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+name: Agent OS check
+about: Test agent compatibility with new OS.
+title: ''
```
Comment (MarcelKemp) [inline .github/ISSUE_TEMPLATE/agent-os-check.md:4]:
```suggestion
title: 'Check wazuh-agent support for OS: <new OS>'
```

**Review by MarcelKemp — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/agent-os-check.md (hunk 1 above)
Comment (mjcr99) [reply .github/ISSUE_TEMPLATE/agent-os-check.md]: Addressed with https://github.com/wazuh/wazuh/pull/23762/commits/100f685b810e4b8fb03d8ed1177ca3a8af25d182

File: .github/ISSUE_TEMPLATE/agent-os-check.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+labels: ''
```
[the remark this answers is not in the document]
Comment (mjcr99) [reply .github/ISSUE_TEMPLATE/agent-os-check.md]: Addressed with https://github.com/wazuh/wazuh/commit/100f685b810e4b8fb03d8ed1177ca3a8af25d182

File: .github/ISSUE_TEMPLATE/agent-os-check.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+|-|-|-|-|-| 
+| **Log collection - System events** | :white_circle: | :white_circle: | :white_circle: | |
+| **Log collection - Log files** | :white_circle: | :white_circle: | :white_circle: | |
+| **Log collection -Command execution** | :white_circle: | :white_circle: | :white_circle: | | 
+| **FIM - Scheduled** | :white_circle: | :white_circle: | :white_circle: | |
+| **FIM - Realtime** | :white_circle: | :black_circle: | :black_circle: | |
+| **FIM - Whodata** | :white_circle: | :black_circle: | :black_circle: | |
+| **SCA** | :white_circle: | :white_circle: | :black_circle: | |
+| **Inventory** | :white_circle: | :white_circle: | :white_circle: | |
+| **Vulnerability Detector** | :white_circle: | :black_circle: | :black_circle: | |
```
[the remark this answers is not in the document]
Comment (mjcr99) [reply .github/ISSUE_TEMPLATE/agent-os-check.md]: Addressed with https://github.com/wazuh/wazuh/commit/100f685b810e4b8fb03d8ed1177ca3a8af25d182

**Review by MarcelKemp — APPROVED** [review]

**Pull request merged**